lib/gems/pending/util/win32/miq-powershell.rb

```diff
@@ -28,7 +28,7 @@ def self.execute(ps_command)
     ps_exec = exepath
     command = "start /wait cmd /c #{ps_exec} #{ps_command}"
     $log.debug "PowerShell: Running command: [#{command}]" if $log
-    ret = MiqUtil.runcmd(command).chomp
+    ret = AwesomeSpawn.run!(command, :combined_output => true).output.chomp
```

Comment:
This one (and the next one) are super-interesting, because there is no way for them to possibly work.  `start` and `taskkill` are Windows commands, and so can't be run by `AwesomeSpawn` nor could they have been run by `MiqUtil.runcmd`.  I converted it for now, but it's actually broken.  I _think_ the correct way to run it is with `WMIHelper.connectServer.runProcess(command, true)`, but I'm not 100% sure and I have no idea how to test it.

Reply:
I believe this code was used a long time ago when we had agents on clients for various reasons.

Searching for MiqPowerShell yields [3 results](https://github.com/search?q=org%3AManageIQ+MiqPowerShell&type=code), only one of which is an [actual caller](https://github.com/ManageIQ/manageiq-providers-scvmm/blob/4165805bf7a9b72476349dcdbaff76eaf56dbbde/app/models/manageiq/providers/microsoft/infra_manager/powershell.rb#L39).  So, I believe we will be able to delete (in a future PR?) most of the guts of MiqPowerShell.  If we try really hard, we can probably delete the whole module as the only remaining use for it is to convert strings. ;-)

Reply:
Yeah, I'm not certain we really need it anymore either...hoping we can kill it in a future PR.